server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -510,6 +512,13 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
         sb.clientAddressSources(cfg.clientAddressSourceList());
         sb.clientAddressTrustedProxyFilter(cfg.trustedProxyAddressPredicate());
 
+        // Config cors policy
```

Comment:
This is optional, maybe it's just me but it feels cleaner to make a simple method call.
I think as-is is fine though

e.g.
```
private static void maybeConfigureCors(ServerBuilder sb, @Nullable CorsConfig corsConfig) {
    if (corsConfig == null || corsConfig.allowedOrigins() == null ||
        corsConfig.allowedOrigins().isEmpty()) {
        return;
    }

    sb.decorator(CorsService.builder(corsConfig.allowedOrigins())
                            .allowRequestMethods(HttpMethod.knownMethods())
                            .allowAllRequestHeaders(true)
                            .allowCredentials()
                            .maxAge(corsConfig.maxAge())
                            .newDecorator());
}
```